### ISSUE TITLE
DJango 3 Redis error: cannot import name 'six'

### ISSUE THREAD

**Issue by maicol07** [WeblateOrg/weblate#3741]
Error during update to 4.0.1:
```
(weblate) maicol07@maicol07:/var/www/html/weblate$ weblate migrate --noinput
Traceback (most recent call last):
  File "/var/www/html/weblate/lib/python3.6/site-packages/django/core/cache/__init__.py", line 50, in _create_cache
    backend_cls = import_string(backend)
  File "/var/www/html/weblate/lib/python3.6/site-packages/django/utils/module_loading.py", line 17, in import_string
    module = import_module(module_path)
  File "/var/www/html/weblate/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/var/www/html/weblate/lib/python3.6/site-packages/django_redis/cache.py", line 8, in <module>
    from .util import load_class
  File "/var/www/html/weblate/lib/python3.6/site-packages/django_redis/util.py", line 8, in <module>
    from django.utils import six
ImportError: cannot import name 'six'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/www/html/weblate/lib/python3.6/site-packages/weblate/vcs/base.py", line 338, in is_supported
    delete_configuration_error(cls.name.lower())
  File "/var/www/html/weblate/lib/python3.6/site-packages/weblate/trans/util.py", line 160, in delete_configuration_erro
r
    errors = cache.get("configuration-errors", [])
  File "/var/www/html/weblate/lib/python3.6/site-packages/django/core/cache/__init__.py", line 98, in __getattr__
    return getattr(caches[DEFAULT_CACHE_ALIAS], name)
  File "/var/www/html/weblate/lib/python3.6/site-packages/django/core/cache/__init__.py", line 79, in __getitem__
    cache = _create_cache(alias)
  File "/var/www/html/weblate/lib/python3.6/site-packages/django/core/cache/__init__.py", line 53, in _create_cache
    "Could not find backend '%s': %s" % (backend, e))
django.core.cache.backends.base.InvalidCacheBackendError: Could not find backend 'django_redis.cache.RedisCache': cannot
 import name 'six'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/www/html/weblate/lib/python3.6/site-packages/django/core/cache/__init__.py", line 50, in _create_cache
    backend_cls = import_string(backend)
  File "/var/www/html/weblate/lib/python3.6/site-packages/django/utils/module_loading.py", line 17, in import_string
    module = import_module(module_path)
  File "/var/www/html/weblate/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/var/www/html/weblate/lib/python3.6/site-packages/django_redis/cache.py", line 8, in <module>
    from .util import load_class
  File "/var/www/html/weblate/lib/python3.6/site-packages/django_redis/util.py", line 8, in <module>
    from django.utils import six
ImportError: cannot import name 'six'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/www/html/weblate/bin/weblate", line 8, in <module>
    sys.exit(main())
  File "/var/www/html/weblate/lib/python3.6/site-packages/weblate/runner.py", line 32, in main
    execute_from_command_line(argv)
  File "/var/www/html/weblate/lib/python3.6/site-packages/django/core/management/__init__.py", line 401, in execute_from
_command_line
    utility.execute()
  File "/var/www/html/weblate/lib/python3.6/site-packages/django/core/management/__init__.py", line 377, in execute
    django.setup()
  File "/var/www/html/weblate/lib/python3.6/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/var/www/html/weblate/lib/python3.6/site-packages/django/apps/registry.py", line 114, in populate
    app_config.import_models()
  File "/var/www/html/weblate/lib/python3.6/site-packages/django/apps/config.py", line 211, in import_models
    self.models_module = import_module(models_module_name)
  File "/var/www/html/weblate/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/var/www/html/weblate/lib/python3.6/site-packages/weblate/addons/models.py", line 45, in <module>
    from weblate.trans.models import Component, Unit
  File "/var/www/html/weblate/lib/python3.6/site-packages/weblate/trans/models/__init__.py", line 33, in <module>
    from weblate.trans.models.component import Component
  File "/var/www/html/weblate/lib/python3.6/site-packages/weblate/trans/models/component.py", line 164, in <module>
    class Component(models.Model, URLMixin, PathMixin):
  File "/var/www/html/weblate/lib/python3.6/site-packages/weblate/trans/models/component.py", line 189, in Component
    choices=VCS_REGISTRY.get_choices(),
  File "/var/www/html/weblate/lib/python3.6/site-packages/weblate/utils/classloader.py", line 106, in get_choices
    for x in sorted(self)
  File "/var/www/html/weblate/lib/python3.6/site-packages/weblate/utils/classloader.py", line 92, in __iter__
    return self.data.__iter__()
  File "/var/www/html/weblate/lib/python3.6/site-packages/django/utils/functional.py", line 48, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/var/www/html/weblate/lib/python3.6/site-packages/weblate/utils/classloader.py", line 71, in data
    return self.load_data()
  File "/var/www/html/weblate/lib/python3.6/site-packages/weblate/vcs/models.py", line 34, in load_data
    if not vcs.is_supported():
  File "/var/www/html/weblate/lib/python3.6/site-packages/weblate/vcs/base.py", line 344, in is_supported
    cls.name, version, cls.req_version, error
  File "/var/www/html/weblate/lib/python3.6/site-packages/weblate/trans/util.py", line 140, in add_configuration_error
    errors = cache.get("configuration-errors", [])
  File "/var/www/html/weblate/lib/python3.6/site-packages/django/core/cache/__init__.py", line 98, in __getattr__
    return getattr(caches[DEFAULT_CACHE_ALIAS], name)
  File "/var/www/html/weblate/lib/python3.6/site-packages/django/core/cache/__init__.py", line 79, in __getitem__
    cache = _create_cache(alias)
  File "/var/www/html/weblate/lib/python3.6/site-packages/django/core/cache/__init__.py", line 53, in _create_cache
    "Could not find backend '%s': %s" % (backend, e))
django.core.cache.backends.base.InvalidCacheBackendError: Could not find backend 'django_redis.cache.RedisCache': cannot
 import name 'six'
```
How can I solve?
Thanks

**Comment by nijel** [comment]
Probably more recent version of django-redis will fix this. Did you update the dependencies on upgrade? Several of them have changed constraints.

**Comment by maicol07** [comment]
Yes, upgrade to 4.0 via pip upgraded dependencies:
```
(weblate) maicol07@maicol07:/var/www/html/weblate$ pip install --upgrade Weblate
Looking in indexes: https://pypi.python.org/simple
Collecting Weblate
  Downloading Weblate-4.0.1.tar.gz (24.7 MB)
     |████████████████████████████████| 24.7 MB 3.6 MB/s
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
Requirement already satisfied, skipping upgrade: django-appconf<1.1,>=1.0.0 in ./lib/python3.6/site-packages (from Webla
te) (1.0.3)
Requirement already satisfied, skipping upgrade: translate-toolkit<2.6.0,>=2.4.0 in ./lib/python3.6/site-packages (from
Weblate) (2.4.0)
Requirement already satisfied, skipping upgrade: pycairo>=1.15.3 in ./lib/python3.6/site-packages (from Weblate) (1.18.1
)
Requirement already satisfied, skipping upgrade: html2text<2020.1.17,>=2019.8.11 in ./lib/python3.6/site-packages (from
Weblate) (2019.9.26)
Collecting Django<3.1,>=3.0.5
  Downloading Django-3.0.5-py3-none-any.whl (7.5 MB)
     |████████████████████████████████| 7.5 MB 227 kB/s
Requirement already satisfied, skipping upgrade: django-redis<4.12.0,>=4.10.0 in ./lib/python3.6/site-packages (from Web
late) (4.10.0)
Collecting translation-finder==2.0
  Downloading translation_finder-2.0-py3-none-any.whl (38 kB)
Requirement already satisfied, skipping upgrade: Cython<0.30,>=0.29.14 in ./lib/python3.6/site-packages (from Weblate) (
0.29.14)
Requirement already satisfied, skipping upgrade: user-agents<2.2,>=2.0 in ./lib/python3.6/site-packages (from Weblate) (
2.0)
Requirement already satisfied, skipping upgrade: python-dateutil>=2.8.1 in ./lib/python3.6/site-packages (from Weblate)
(2.8.1)
Collecting jsonschema<3.3.0,>=3.0.0
  Downloading jsonschema-3.2.0-py2.py3-none-any.whl (56 kB)
     |████████████████████████████████| 56 kB 545 kB/s
Requirement already satisfied, skipping upgrade: GitPython<3.2.0,>=2.1.15 in ./lib/python3.6/site-packages (from Weblate
) (3.0.5)
Requirement already satisfied, skipping upgrade: requests<2.24.0,>=2.20.0 in ./lib/python3.6/site-packages (from Weblate
) (2.22.0)
Requirement already satisfied, skipping upgrade: borgbackup<1.2.0,>=1.1.9 in ./lib/python3.6/site-packages (from Weblate
) (1.1.10)
Requirement already satisfied, skipping upgrade: social-auth-app-django==3.1.0 in ./lib/python3.6/site-packages (from We
blate) (3.1.0)
Collecting djangorestframework<3.12.0,>=3.11.0
  Downloading djangorestframework-3.11.0-py3-none-any.whl (911 kB)
     |████████████████████████████████| 911 kB 1.8 MB/s
Requirement already satisfied, skipping upgrade: hiredis<1.1.0,>=1.0.1 in ./lib/python3.6/site-packages (from Weblate) (
1.0.1)
Collecting social-auth-core==3.3.3
  Downloading social_auth_core-3.3.3-py3-none-any.whl (326 kB)
     |████████████████████████████████| 326 kB 3.2 MB/s
Requirement already satisfied, skipping upgrade: filelock<3.1.0,>=3.0.0 in ./lib/python3.6/site-packages (from Weblate)
(3.0.12)
Requirement already satisfied, skipping upgrade: diff-match-patch==20181111 in ./lib/python3.6/site-packages (from Webla
te) (20181111)
Requirement already satisfied, skipping upgrade: jellyfish<0.8.0,>=0.7.2 in ./lib/python3.6/site-packages (from Weblate)
 (0.7.2)
Requirement already satisfied, skipping upgrade: pygobject>=3.27.0 in ./lib/python3.6/site-packages (from Weblate) (3.34
.0)
Requirement already satisfied, skipping upgrade: openpyxl!=3.0.2,<3.1,>=2.6.0 in ./lib/python3.6/site-packages (from Web
late) (2.6.3)
Requirement already satisfied, skipping upgrade: sentry-sdk<0.15.0,>=0.13.0 in ./lib/python3.6/site-packages (from Webla
te) (0.14.1)
Requirement already satisfied, skipping upgrade: misaka<2.2.0,>=2.1.0 in ./lib/python3.6/site-packages (from Weblate) (2
.1.1)
Requirement already satisfied, skipping upgrade: celery[redis]!=4.4.1,<4.5,>=4.2.0 in ./lib/python3.6/site-packages (fro
m Weblate) (4.3.0)
Requirement already satisfied, skipping upgrade: bleach<3.2.0,>=3.1.1 in ./lib/python3.6/site-packages (from Weblate) (3
.1.1)
Requirement already satisfied, skipping upgrade: setuptools>=36.0.1 in ./lib/python3.6/site-packages (from Weblate) (41.
2.0)
Collecting weblate-schemas==0.3
  Downloading weblate_schemas-0.3-py3-none-any.whl (6.0 kB)
Requirement already satisfied, skipping upgrade: Whoosh==2.7.4 in ./lib/python3.6/site-packages (from Weblate) (2.7.4)
Requirement already satisfied, skipping upgrade: Pillow<7.2.0,>=6.0.0 in ./lib/python3.6/site-packages (from Weblate) (6
.2.1)
Requirement already satisfied, skipping upgrade: kombu[redis]!=4.6.4,!=4.6.5,>=4.6.0 in ./lib/python3.6/site-packages (f
rom Weblate) (4.6.7)
Requirement already satisfied, skipping upgrade: siphashc<2.0,>=1.2 in ./lib/python3.6/site-packages (from Weblate) (1.2
)
Collecting django-compressor<2.5,>=2.4
  Downloading django_compressor-2.4-py2.py3-none-any.whl (126 kB)
     |████████████████████████████████| 126 kB 4.4 MB/s
Requirement already satisfied, skipping upgrade: lxml!=4.3.1,>=4.2 in ./lib/python3.6/site-packages (from Weblate) (4.4.
1)
Collecting django-crispy-forms<1.10.0,>=1.9.0
  Downloading django_crispy_forms-1.9.0-py2.py3-none-any.whl (107 kB)
     |████████████████████████████████| 107 kB 3.5 MB/s
Requirement already satisfied, skipping upgrade: six in ./lib/python3.6/site-packages (from django-appconf<1.1,>=1.0.0->
Weblate) (1.14.0)
Requirement already satisfied, skipping upgrade: pytz in ./lib/python3.6/site-packages (from Django<3.1,>=3.0.5->Weblate
) (2019.2)
Collecting asgiref~=3.2
  Downloading asgiref-3.2.7-py2.py3-none-any.whl (19 kB)
Requirement already satisfied, skipping upgrade: sqlparse>=0.2.2 in ./lib/python3.6/site-packages (from Django<3.1,>=3.0
.5->Weblate) (0.3.0)
Requirement already satisfied, skipping upgrade: redis>=2.10.0 in ./lib/python3.6/site-packages (from django-redis<4.12.
0,>=4.10.0->Weblate) (3.3.8)
Requirement already satisfied, skipping upgrade: chardet in ./lib/python3.6/site-packages (from translation-finder==2.0-
>Weblate) (3.0.4)
Requirement already satisfied, skipping upgrade: ruamel.yaml in ./lib/python3.6/site-packages (from translation-finder==
2.0->Weblate) (0.16.5)
Requirement already satisfied, skipping upgrade: ua-parser>=0.8.0 in ./lib/python3.6/site-packages (from user-agents<2.2
,>=2.0->Weblate) (0.8.0)
Collecting pyrsistent>=0.14.0
  Downloading pyrsistent-0.16.0.tar.gz (108 kB)
     |████████████████████████████████| 108 kB 4.6 MB/s
Requirement already satisfied, skipping upgrade: importlib-metadata; python_version < "3.8" in ./lib/python3.6/site-pack
ages (from jsonschema<3.3.0,>=3.0.0->Weblate) (0.23)
Requirement already satisfied, skipping upgrade: attrs>=17.4.0 in ./lib/python3.6/site-packages (from jsonschema<3.3.0,>
=3.0.0->Weblate) (19.1.0)
Requirement already satisfied, skipping upgrade: gitdb2>=2.0.0 in ./lib/python3.6/site-packages (from GitPython<3.2.0,>=
2.1.15->Weblate) (2.0.6)
Requirement already satisfied, skipping upgrade: idna<2.9,>=2.5 in ./lib/python3.6/site-packages (from requests<2.24.0,>
=2.20.0->Weblate) (2.8)
Requirement already satisfied, skipping upgrade: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in ./lib/python3.6/site-package
s (from requests<2.24.0,>=2.20.0->Weblate) (1.25.5)
Requirement already satisfied, skipping upgrade: certifi>=2017.4.17 in ./lib/python3.6/site-packages (from requests<2.24
.0,>=2.20.0->Weblate) (2019.9.11)
Requirement already satisfied, skipping upgrade: oauthlib>=1.0.3 in ./lib/python3.6/site-packages (from social-auth-core
==3.3.3->Weblate) (3.1.0)
Requirement already satisfied, skipping upgrade: defusedxml>=0.5.0rc1; python_version >= "3.0" in ./lib/python3.6/site-p
ackages (from social-auth-core==3.3.3->Weblate) (0.6.0)
Requirement already satisfied, skipping upgrade: python3-openid>=3.0.10; python_version >= "3.0" in ./lib/python3.6/site
-packages (from social-auth-core==3.3.3->Weblate) (3.1.0)
Requirement already satisfied, skipping upgrade: PyJWT>=1.4.0 in ./lib/python3.6/site-packages (from social-auth-core==3
.3.3->Weblate) (1.7.1)
Requirement already satisfied, skipping upgrade: requests-oauthlib>=0.6.1 in ./lib/python3.6/site-packages (from social-
auth-core==3.3.3->Weblate) (1.2.0)
Collecting cryptography>=1.4
  Downloading cryptography-2.9-cp35-abi3-manylinux2010_x86_64.whl (2.7 MB)
     |████████████████████████████████| 2.7 MB 2.5 MB/s
Requirement already satisfied, skipping upgrade: et-xmlfile in ./lib/python3.6/site-packages (from openpyxl!=3.0.2,<3.1,
>=2.6.0->Weblate) (1.0.1)
Requirement already satisfied, skipping upgrade: jdcal in ./lib/python3.6/site-packages (from openpyxl!=3.0.2,<3.1,>=2.6
.0->Weblate) (1.4.1)
Requirement already satisfied, skipping upgrade: cffi>=1.0.0 in ./lib/python3.6/site-packages (from misaka<2.2.0,>=2.1.0
->Weblate) (1.13.2)
Requirement already satisfied, skipping upgrade: vine>=1.3.0 in ./lib/python3.6/site-packages (from celery[redis]!=4.4.1
,<4.5,>=4.2.0->Weblate) (1.3.0)
Requirement already satisfied, skipping upgrade: billiard<4.0,>=3.6.0 in ./lib/python3.6/site-packages (from celery[redi
s]!=4.4.1,<4.5,>=4.2.0->Weblate) (3.6.1.0)
Requirement already satisfied, skipping upgrade: webencodings in ./lib/python3.6/site-packages (from bleach<3.2.0,>=3.1.
1->Weblate) (0.5.1)
Requirement already satisfied, skipping upgrade: amqp<2.6,>=2.5.2 in ./lib/python3.6/site-packages (from kombu[redis]!=4
.6.4,!=4.6.5,>=4.6.0->Weblate) (2.5.2)
Requirement already satisfied, skipping upgrade: rjsmin==1.1.0 in ./lib/python3.6/site-packages (from django-compressor<
2.5,>=2.4->Weblate) (1.1.0)
Requirement already satisfied, skipping upgrade: rcssmin==1.0.6 in ./lib/python3.6/site-packages (from django-compressor
<2.5,>=2.4->Weblate) (1.0.6)
Requirement already satisfied, skipping upgrade: ruamel.yaml.clib>=0.1.2; platform_python_implementation == "CPython" an
d python_version < "3.8" in ./lib/python3.6/site-packages (from ruamel.yaml->translation-finder==2.0->Weblate) (0.1.2)
Requirement already satisfied, skipping upgrade: zipp>=0.5 in ./lib/python3.6/site-packages (from importlib-metadata; py
thon_version < "3.8"->jsonschema<3.3.0,>=3.0.0->Weblate) (0.6.0)
Requirement already satisfied, skipping upgrade: smmap2>=2.0.0 in ./lib/python3.6/site-packages (from gitdb2>=2.0.0->Git
Python<3.2.0,>=2.1.15->Weblate) (2.0.5)
Requirement already satisfied, skipping upgrade: pycparser in ./lib/python3.6/site-packages (from cffi>=1.0.0->misaka<2.
2.0,>=2.1.0->Weblate) (2.19)
Requirement already satisfied, skipping upgrade: more-itertools in ./lib/python3.6/site-packages (from zipp>=0.5->import
lib-metadata; python_version < "3.8"->jsonschema<3.3.0,>=3.0.0->Weblate) (7.2.0)
Building wheels for collected packages: Weblate, pyrsistent
  Building wheel for Weblate (PEP 517) ... done
  Created wheel for Weblate: filename=Weblate-4.0.1-py3-none-any.whl size=23267987 sha256=6f7ce47346a548d111260e5906a28e
1adbbe36edfc59a26dcb5403a2a70e2e3b
  Stored in directory: /home/maicol07/.cache/pip/wheels/71/e5/17/b8af032fad25a62f7796fb3f60d57738b43a9afe9a85f811d4
  Building wheel for pyrsistent (setup.py) ... done
  Created wheel for pyrsistent: filename=pyrsistent-0.16.0-cp36-cp36m-linux_x86_64.whl size=97741 sha256=393583ad7cf73d8
78cbfee329df91e8fe7ab671a8702c58c205c8d66eb721b9c
  Stored in directory: /home/maicol07/.cache/pip/wheels/d1/8a/1c/32ab9017418a2c64e4fbaf503c08648bed2f8eb311b869a464
Successfully built Weblate pyrsistent
Installing collected packages: asgiref, Django, translation-finder, pyrsistent, jsonschema, djangorestframework, cryptog
raphy, social-auth-core, weblate-schemas, django-compressor, django-crispy-forms, Weblate
  Attempting uninstall: Django
    Found existing installation: Django 2.2.5
    Uninstalling Django-2.2.5:
      Successfully uninstalled Django-2.2.5
  Attempting uninstall: translation-finder
    Found existing installation: translation-finder 1.8
    Uninstalling translation-finder-1.8:
      Successfully uninstalled translation-finder-1.8
  Attempting uninstall: djangorestframework
    Found existing installation: djangorestframework 3.10.3
    Uninstalling djangorestframework-3.10.3:
      Successfully uninstalled djangorestframework-3.10.3
  Attempting uninstall: social-auth-core
    Found existing installation: social-auth-core 3.2.0
    Uninstalling social-auth-core-3.2.0:
      Successfully uninstalled social-auth-core-3.2.0
  Attempting uninstall: django-compressor
    Found existing installation: django-compressor 2.3
    Uninstalling django-compressor-2.3:
      Successfully uninstalled django-compressor-2.3
  Attempting uninstall: django-crispy-forms
    Found existing installation: django-crispy-forms 1.8.1
    Uninstalling django-crispy-forms-1.8.1:
      Successfully uninstalled django-crispy-forms-1.8.1
  Attempting uninstall: Weblate
    Found existing installation: Weblate 3.11.3
    Uninstalling Weblate-3.11.3:
      Successfully uninstalled Weblate-3.11.3
Successfully installed Django-3.0.5 Weblate-4.0.1 asgiref-3.2.7 cryptography-2.9 django-compressor-2.4 django-crispy-for
ms-1.9.0 djangorestframework-3.11.0 jsonschema-3.2.0 pyrsistent-0.16.0 social-auth-core-3.3.3 translation-finder-2.0 web
late-schemas-0.3
```

**Comment by nijel** [comment]
What version of django_redis do you have installed?

**Comment by maicol07** [comment]
4.10.0

**Comment by nijel** [comment]
Ah, it seems that 4.11.0 is actually needed.

**Comment by nijel** [comment]
I've adjusted our test to catch this kind of error and it's now failing with same error as you have, so upgrading should indeed address this.

https://github.com/WeblateOrg/weblate/runs/592499974?check_suite_focus=true

**Comment by nijel** [comment]
Fixed dependencies in d92d8abcbb3412c5c4bfa0dea086b29a043d81b3
